README.md

```diff
@@ -14,10 +14,11 @@ Memcached protocol 'wrapper' for Tarantool.
 
  * Tarantol 1.6.8+ with header files (tarantool && tarantool-dev packages).
  * Cyrus SASL library (with header files)
- * Python >= 2.7, <3 with next packages (for testing only):
+ * Python >= 2.7 with next packages (for testing only):
    - PyYAML
    - msgpack-python
    - six==1.9.0
+   - gevent==1.1b5
```

Comment:
This list is not clear for me. At least it is not based on actual test-run's `requirements.txt`: we don't depend on external `msgpack-python` (it is bundled into test-run), we don't depend on `six`, we have `gevent==21.*` (but any gevent version fit good for test-run in fact if it works on given Python version).

Reply:
Fixed:

```diff
--- a/README.md
+++ b/README.md
@@ -16,10 +16,8 @@ Memcached protocol 'wrapper' for Tarantool.
  * Tarantol 1.6.8+ with header files (tarantool && tarantool-dev packages).
  * Cyrus SASL library (with header files)
  * Python >= 3.7 with next packages (for testing only):
-   - PyYAML
-   - msgpack-python
-   - six==1.9.0
-   - gevent==1.1b5
+   - PyYAML 5+
+   - gevent 21+
 
 ### Installation
 
```